catalog/gitops/hydration-trigger.yaml

```diff
@@ -57,7 +60,7 @@ spec:
           - |
             set -e
             git pull origin $BRANCH_NAME || true # Ignore errors in case branch doesn't exist
-            rm -rf /deployment-workspace/config/*
+            git rm -rf /deployment-workspace/config/*
```

Comment:
Wouldn't this fail on a fresh repo if $BRANCH_NAME does not exist on remote?

Reply:
We explicitly create the config directory [here](https://github.com/GoogleCloudPlatform/blueprints/pull/41/files#diff-ff2072c72360e196e595d3ba0dd5a47e790d0a6539a58224043e4b5a751a00c4R26).